### ISSUE TITLE
Change recovery codes filename to reflect testpypi or pypi

### ISSUE THREAD

**Issue by diegoe** [pypa/warehouse#9190]
**What's the problem this feature will solve?**
Currently if you generate recovery codes for testpypi and pypi, both files are suggested for download as `PyPI-Recovery-Codes.txt`.

**Describe the solution you'd like**
The suggested filename reflects the origin. Example: `TestPyPi-Recovery-Codes.txt` and `PyPi-Recovery-Codes.txt`.

**Additional context**
It should be possible to change it to an if/else to set a prefix at:
https://github.com/pypa/warehouse/blob/main/warehouse/templates/manage/account/recovery_codes-provision.html#L64-L66

**Comment by ewdurbin** [comment]
I'm not positive, but the `testPyPI` variable may even be accessible since the template for the recovery codes extends from manage_base which extends from base.

**Comment by di** [comment]
> I'm not positive, but the `testPyPI` variable may even be accessible since the template for the recovery codes extends from manage_base which extends from base.

Yes, this should be true!

I've added the "good first issue" label to this issue.

**Comment by abid1998** [comment]
@di I can try this issue if you can give me more context.

**Comment by AustinTSchaffer** [comment]
@abid1998 I'm not di, but commenting for additional context:

If you set up an account, verify your email address, and set up 2-factor authentication, the "Account Settings" menu gives you an additional option for generating/regenerating account recovery codes. These codes can be used to get access to your account in the event your 2-factor authentication device becomes inaccessible, gets lost, breaks, etc. Once you generate some codes, you have the option to download them as a text file.

![image](https://user-images.githubusercontent.com/16184219/116791758-73351700-aa8a-11eb-8b6e-28933a9f221a.png)

(I generated these codes running warehouse on localhost, but figured I should partially black them out anyway.)

This would be changing the default name for that text file to "TestPyPI-Recovery-Codes.txt" when you're on `test.pypi.org`, as opposed to "PyPI-Recovery-Codes.txt" when you're on `pypi.org`, probably to reduce confusion and prevent users from overwriting their actual recovery codes when using `test.pypi.org`.

Hope this helps! I played around with this a bit and ended up coming up with a solution. Putting in a pull request soon.
